src/Styles/ContactPage.css

```diff
@@ -1,32 +1,103 @@
-.contactUs h1,h2{
+body{
```

Comment:
Please change this into a page-specific component (i.e: add an id or className for the container or div). Using `body` directly will cause all the pages on the website to have this background:
![Screen Shot 2022-06-18 at 5 36 03 PM](https://user-images.githubusercontent.com/49298813/174457703-3a30b686-8430-4225-893e-0800468cb5a7.png)